hs_core/hydroshare/utils.py

```diff
@@ -459,6 +463,8 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id, set_to_private=False):
         if value:
             if avu_name == 'isPublic' and set_to_private:
                 istorage.setAVU(dest_coll, avu_name, 'False')
+            elif avu_name == 'quotaUserName' and requesting_user:
+                istorage.setAVU(dest_coll, avu_name, requesting_user.username)
```

Comment:
But the copy_resource_files_and_AVUs is called **after** resource creation. Theoretically, at creation, the quota is set. This is a creation-level task.